### PR TITLE
test(perf_with_ops): add new test to run on i3en

### PR DESCRIPTION
**Pull request by fgelcer**
we want to start benchmarking runs on i3en.2xlarge
instances, and we aim to do in a separated test.
Adding the needed files in here, so we can create
a new job in Jenkins to call it.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: jenkins-pipelines/perf-regression-latency-250gb-with-nemesis-i3en.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression-latency-250gb-with-nemesis.yaml", "configurations/aws/i3en_2xlarge.yaml"]''',
+    sub_tests: ["test_latency_write_with_nemesis", "test_latency_read_with_nemesis", "test_latency_mixed_with_nemesis"],
+
+    timeout: [time: 1600, unit: "MINUTES"]
```
Comment (aleksbykov) [inline jenkins-pipelines/perf-regression-latency-250gb-with-nemesis-i3en.jenkinsfile:12]: i think, it is not needed any more

**Review by aleksbykov — DISMISSED** [review]
LGTM

File: jenkins-pipelines/perf-regression-latency-250gb-with-nemesis-i3en.jenkinsfile (hunk 1 above)
Comment (fgelcer) [reply jenkins-pipelines/perf-regression-latency-250gb-with-nemesis-i3en.jenkinsfile]: removed

**Review by roydahan — APPROVED** [review]

**Comment by fgelcer** [comment]
only the main commit was backported, and only to `branch-perf-v12`

still need to backport it to `branch-perf-v13`

**Comment by roydahan** [comment]
v13 isn't relevant for it.

**Comment by fgelcer** [comment]
> v13 isn't relevant for it.

no, but we must keep it updated, so it will fall behind if we decide to make any change in that branch

**Pull request merged**